cypress/integration/manualTests/monitor.ts

```diff
@@ -50,3 +52,27 @@ describe('Monitor homepage links', () => {
     });
   });
 });
+
+describe('Test launch button icons', () => {
+  it('find a WDL workflow', () => {
+    cy.visit('/search');
+    cy.wait(10000); // wait for the tabs to become clickable
+    goToTab('Workflows');
+
+    // click to sort by descriptor type descending so WDL is at the top
+    cy.get('[data-cy=descriptorType]')
+      .click()
```

Comment:
what's the purpose of the double click?

Reply:
the first click sorts by descriptor type ascending, the second click reverses the sort so that the first workflow is WDL